site/docs/v1/tech/core-concepts/users.adoc

```diff
@@ -35,11 +35,15 @@ FusionAuth includes reporting on the number of daily and monthly active users. W
 * A JWT is refreshed using a Refresh Token.
 * SSO is used; this calls the login ping.
 
+[NOTE.note]
+====
+Users imported with the link:/docs/v1/tech/apis/users/#import-users[Import Users API] do not count as a monthly active user (MAU).
```

Comment:
What you have is fine, but a suggestion:

Since the sentence begins with "Users".... `Import Users API` -> `Import API`.